docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+mysql:
+    extends:
+      file: common-compose.yml
+      service: mysql
+    links:
+      - consul:consul
+
+mysql_replica:
```

Comment:
> restart a replica, then `docker exec` a command to the replicas to `CHANGE MASTER` to the new primary

Would Containerbuddy be able to detect the changed master? Not saying we should, but we could automatically trigger the `CHANGE MASTER` in that context, yes?